frog/imports/api/activities.js

```diff
@@ -105,8 +116,8 @@ export const copyActivityIntoGraphActivity = (
 };
 
 export const copyOperatorIntoGraphOperator = (
-  graphOperatorId,
-  fromOperatorId
+  graphOperatorId: string,
+  fromOperatorId: string
```

Comment:
I am eagerly waiting for the day Flow allows nominal typing, so that we can say that this field takes an operatorId, and Flow will complain if you try to put a GraphId there... :)